File: quotes/src/main/java/fun/apps/quotes/model/Quote.java
package fun.apps.quotes.model;

import java.io.Serializable;
import java.util.Collection;

import lombok.Builder;
import lombok.Data;

@Builder
@Data
public class Quote implements Serializable{
	private static final long serialVersionUID = -6499866882889134213L;
	private String _id;
	private String quotation; 
	private String author_id;
	private String informationsource_id;
	private Collection<Note> notes;
}

File: quotes-ui/src/app/shared/model/quote.ts
import { Author } from './author';
import { InformationSource } from './information-source';
import { Note } from './note';

export class Quote {
  quotation: string;
  author_id: string;
  author: Author;
  informationsource_id: string;
  source: InformationSource;
  notes: Note[];

  constructor(
    quotation: string,
    author_id: string,
    informationsource_id: string
  ){
    this.quotation = quotation;
    this.author_id = author_id;
    this.informationsource_id = informationsource_id;
  }


}
